packages/expo-updates/android/src/main/java/expo/modules/updates/manifest/raw/RawManifest.kt

```diff
@@ -21,6 +21,11 @@ abstract class RawManifest(protected val json: JSONObject) {
   @Deprecated(message = "Prefer to use specific field getters")
   fun getRawJson(): JSONObject = json
 
+  @Deprecated(message = "toString should be called on the underlying getRawJson instead")
+  override fun toString(): String {
+    throw Error("toString called on RawManifest")
```

Comment:
Curious why you don't want to just implement this as `return getRawJson().toString()`; to me it would make sense to call `toString` on a manifest object if I wanted a stringified version, rather than needing to think about how the RawManifest class stores the underlying JSON object (which seems like an implementation detail).